docs/src/content/docs/hi/404.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Not found
```

Comment:
Thanks for your contribution 🙌 

Small question: the title can be safely translated, would it make sense or improve the experience to translate it in Hindi too?

Reply:
of course, I completely omitted title at first but with latest commit that should be resolved)